src/Session/FlashGlobals.js

```diff
@@ -12,11 +12,11 @@
 const _ = require('lodash')
 
 module.exports = function (View) {
-  View.global('old', function (key, defaultValue) {
+  View.global('old', function (key, defaultValue = '') {
     return _.get(this.resolve('flashMessages'), key, defaultValue)
   })
 
-  View.global('flashMessage', function (key, defaultValue) {
+  View.global('flashMessage', function (key, defaultValue = '') {
```

Comment:
A final change, we don't need an empty string here, since `old` method will take care of it for us.

Reply:
Any news about this change @lffg? 😄

Reply:
No, I think it's okay. If you need anything, you can tell.

Reply:
Can you share why it's okay? All this method does it calls an internal method, which has it's own default.

Reply:
Yes. You're right. I didn't noticed that. Sorry for that. _(I'm new to this type of contribution)._
I'll change that.

Reply:
By the way, do you prefer the default value to be in the "`old`" or "`flashMessage`" global? I think in "`old`" would be better, since the initial suggestion of this pull request was to add the default value (empty string) only to the "`old`" global.

What do you think?

Thanks,

Reply:
Should definitively be in the `old` method.